### PR TITLE
Update libraries for more recent react work

### PR DESCRIPTION
**Pull request by mbonneau**

### CODE REVIEW

File: src/Server.php [hunk 1]
```diff
@@ -23,30 +25,25 @@ class Server extends Observable
...
+        $this->bindAddress      = $bindAddressOrPort;
         $this->useMessageObject = $useMessageObject;
         $this->subProtocols     = $subProtocols;
         $this->loop             = $loop ?: \EventLoop\getLoop();
     }
 
     public function _subscribe(ObserverInterface $observer): DisposableInterface
     {
-        $socket = new \React\Socket\Server($this->loop);
+        $socket = new \React\Socket\Server($this->bindAddress, $this->loop);
```
Comment (davidwdan) [inline src/Server.php]: Maybe do a `use \React\Socket\Server as SocketServer`?

**Comment by WyriHaximus** [comment]
Cool! :+1: Does the public API stay the same for `2.0`?

**Comment by davidwdan** [comment]
The client API stayed the same, but the server changed a little.

**Comment by WyriHaximus** [comment]
Cheers, thanks :+1: . (That makes updating `api-clients/pusher` to support both a breeze.)

**Comment by davidwdan** [comment]
There is actually one client api change that does effect pusher:
https://github.com/RxPHP/RxWebsocket/pull/13/commits/45d38511032456cc96ba3e84ed97d0cc0456731c?diff=unified#diff-ff73e042e738204c6da009e2ed19f783R26

**Comment by WyriHaximus** [comment]
Hmm we could work around that tbh. Will thinker up a PR 👍

**Pull request merged**